src/main/java/pl/byczazagroda/trackexpensesappbackend/model/Wallet.java

```diff
@@ -45,8 +45,8 @@ public class Wallet implements Serializable {
 
     private Instant creationDate;
 
-    public Wallet(String name) {
+    public Wallet(String name, Instant creationDate) {
```

Comment:
We should stick with the previous, one parameter constructor (only String name argument).
We do not want the client to send us creation date value via request while creating or updating wallet.
Creation Date field should be also generated automatically by the server (via `Instant.now()` method.

Reply:
Ok but it was told to stick to convention used in task 23, already merged to branch develop. There is used convention with two-args constructor. So why should I roll back to one-arg constructor? I see some misaligment here... Can you Please clarify it?

Reply:
Take  a look at `Wallet.java` in develop branch. There is only one-parameter constructor with String name argument

![image](https://user-images.githubusercontent.com/72097324/196003266-748e7e21-838c-401f-b8ca-abc841ec5f38.png)

Reply:
OK, updated.